src/sensor_mag_plugin.cpp

```diff
@@ -122,4 +144,4 @@ void SensorMagPlugin::addNoise(Eigen::Vector3d* magnetic_field, const double dt)
     _bias[i] = phi_d * _bias[i] + sigma_b_d * _standard_normal_distribution(_random_generator);
     (*magnetic_field)[i] = (*magnetic_field)[i] + _bias[i] + sigma_d * _standard_normal_distribution(_random_generator);
   }
-}
+}
```

Comment:
Could you add a EOF just to make it more clean?

Reply:
Done.